src/queries/rum-common.sql

```diff
@@ -259,6 +259,58 @@ VALUES
     inreadonly);
 END
 
+CREATE OR REPLACE PROCEDURE helix_reporting.UPDATE_DOMAIN_INFO(
```

Comment:
You are only using the procedures in one single query, therefore I'd keep it entirely in the query instead of indirecting it in `rum-common`

Reply:
I'll try that again.  My first effort was exactly that but could not pass sqlfluff on my local VS Code.  I did not attempt to commit, maybe it will pass on build.

Reply:
Well, I tested after commit and it works but the PR is complaining about sqlfluff "unparsable section".  Please advise.

Reply:
sqlfluff can't parse the procedure stuff, therefore I'm trying to use it as little as possible, and when it can't be avoided, I'm extra careful.